target/linux/ramips/image/mt7621.mk

```diff
@@ -86,6 +86,11 @@ define Build/mitrastarimage
 	mv $@.new $@
 endef
 
+define Build/sge-uimage
+	uimage_padhdr -i $@ -o $@.new -l 96
+	mv $@.new $@
+endef
```

Comment:
I wonder whether it's really necessary to define that again for every different padding required.
I'd consider to just merge that with mitrastarimage and fonfxcimage in image/Makefile.

I will create a patch for that separately ...

Reply:
https://patchwork.ozlabs.org/project/openwrt/patch/20200710121623.1923-1-freifunk@adrianschmutzler.de/

With that, the sge-uimage recipe definition could be dropped and replaced by `uimage-padhdr 96` for the image build recipes.

Reply:
@Sixtiz I've pushed my patch now, so you can rebase onto it.

Reply:
Please rebase and use that generic `uimage-padhdr 96` as suggested.